mobile/docs/root/api/starting_envoy.rst

```diff
@@ -527,6 +527,60 @@ Note that Envoy will fail to start up in debug mode if an unknown guard is speci
   // Swift
   builder.setRuntimeGuard("feature", true)
 
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+``addRtdsLayer``
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Adds an RTDS layer to the bootstrap configuration.
+Requires that ADS be configured via `setAggregatedDiscoveryService()`.
+See the following link for details:
+https://www.envoyproxy.io/docs/envoy/latest/api-v3/service/runtime/v3/rtds.proto
+
+**Example**::
+
+  // Kotlin
+  builder.addRtdsLayer("rtds_layer_name", 10)
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+``setAggregatedDiscoveryService``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Adds ADS to bootstrap configuration, for instance to be used with RTDS and CDS layers.
+Optional params allow configuring a JWT token and SSL. See the following link for details:
+https://www.envoyproxy.io/docs/envoy/latest/configuration/overview/xds_api#config-overview-ads
+
+**Example**::
+
+  // Kotlin
+  builder.setAggregatedDiscoveryService("192.168.1.1", 0)
```

Comment:
It's unclear (to me anyway) what the parameters do here. I'm assuming that the first parameter must be an IPv4 address where the ADS is hosted? I can't really guess what the `0` signifies since there's no parameter label and I can't tell what it could be based on the type. The paragraph above mentions that there are optional params for configuring a JWT and SSL, but I don't know if the `0` is related to that or if those params are omitted from this example.

Reply:
Tried adding more thorough documentation, had to come up with some new formatting style.  Let me know what you think!